gui.py

```diff
@@ -0,0 +1,284 @@
+import sys
+
+
+from PyQt5 import QtCore, QtGui, QtWidgets
+from PyQt5.QtCore import Qt, QSettings
+from discodos.qt.MainWindow import Ui_MainWindow
+from discodos.models import Mix, log, Collection
+from discodos.config import Config
+from discodos.utils import is_number
+
+
+class GuiTableViewModel(QtGui.QStandardItemModel):
+#class MyModel(QtCore.QAbstractTableModel):
```

Comment:
remove this commented out line or if you wanna keep it, add a space after the hash so it reflects what PEP8 linters want: `# foo`